packages/gatsby-plugin-sitemap/src/__tests__/gatsby-node.js

```diff
@@ -176,5 +177,16 @@ describe(`Test plugin sitemap`, async () => {
       expect(filePath).toEqual(path.join(`public`, `sitemap.xml`))
       expect(contents).toMatchSnapshot()
     })
+    it(`should include path prefix when creating creating index sitemap`, async () => {
+      const sitemapSpy = jest.spyOn(sitemap, `createSitemapIndex`)
+      const options = {
+        sitemapSize: 1,
+      }
+      const prefix = `/test`
+      await onPostBuild({ graphql, pathPrefix: prefix }, options)
+      expect(sitemapSpy.mock.calls[0][0].hostname).toEqual(
```

Comment:
NIT: don't need to change this (we do use it in different places as well), but for future, please use
```js
expect(sitemapSpy).toBeCalledWith(
  expect.objectContaining({
    hostname: `${queryResult.data.site.siteMetadata.siteUrl}${prefix}`
  })
)
```
reaching to `.mock.calls[0][0]` works, but is less readable